src/prettier-comments/language-js/comments.js

```diff
@@ -7,8 +7,8 @@ const {
     addDanglingComment,
     getNextNonSpaceNonCommentCharacterIndex
   }
-} = require('prettier');
-const privateUtil = require("../common/util");
+} = require('prettier/standalone');
+const privateUtil = require('../common/util');
```

Comment:
can we keep the `"` in this file as it was just copied pasted from the prettier main repo?

Reply:
done